portfolio/src/main/java/com/google/sps/servlets/CommentsServlet.java

```diff
@@ -54,7 +61,19 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
         response.sendRedirect("/comments.html");
     }
     private Comment getComment(HttpServletRequest request) {
-        Comment comment = new Comment(request.getParameter(NAME_PARAMETER), request.getParameter(COMMENT_PARAMETER));
+        long timestamp = System.currentTimeMillis();
+        Comment comment = new Comment(request.getParameter(NAME_PARAMETER), request.getParameter(COMMENT_PARAMETER), timestamp);
+        newEntity(comment);
         return comment;
     }
+    private void newEntity(Comment comment) {
+        Entity task = new Entity("Comment");
+        task.setProperty("name", comment.getName());
+        task.setProperty("message", comment.getMessage());
+        task.setProperty("timestamp", comment.getTimestamp());
```

Comment:
I would take these hardcoded strings, like `"name"` or `"message"` or `"timestamp"`, and store them in a `private static final` variable

Reply:
This is so you can easily change only a single line in the file (if these need to be changed) rather than doing a find/replace